### DISCUSSION TITLE
能否提供触控板双手滑动前进后退和鼠标按钮前进后退，类似网页端的体验

### DISCUSSION THREAD

**Discussion by LeeeSe** [tw93/Pake#104] [General]
经测试无法在app中利用鼠标按钮和触控板快速前进和后退
希望能够支持

**Comment by tw93** [comment]
主要是哪一个App 方便说一下名称吗？我拿着那个去测试

**Comment by LeeeSe** [comment]
v2ex.app
reference.app
系统：MacOS 13.1 Beta版

**Comment by tw93** [comment]
[Reference_1.0.0_universal.dmg.zip](https://github.com/tw93/Pake/files/10090940/Reference_1.0.0_universal.dmg.zip)
[V2ex_1.0.0_universal.dmg.zip](https://github.com/tw93/Pake/files/10090942/V2ex_1.0.0_universal.dmg.zip)
已经支持，整体试了试效果还可以，你也可以试试

**Reply by Fire-fier** [reply]
这两个有windows的版本吗？

**Comment by LiXin-x** [comment]
一些以前打包的应用也希望支持一下，比如微信读书。体验和 Safari 一致，就很棒

**Reply by tw93** [reply]
这一块最近我们会发一个1.0版本，基本上所有的就都支持了
你着急的话可以自己用命令行一键打包，可以去看readme文档哦

**Reply by LiXin-x** [reply]
已打包。

期待 Pake 1.0 能有
* 缩小放大页面的手势
* 跳转链接到浏览器
* 窗口大小记忆

**Reply by tw93** [reply]
1. 可以通command+ +- 会更合适
2. 现在其实是支持的，不过需要配置下，具体是哪一个app哦？
3. 这个有计划

**Reply by LiXin-x** [reply]
1. 缩小放大页面的手势，会用作比如网页应用本身不支持放大图片，直接放大页面查看
2. 微信读书要打开「传书到手机」，这原本会新开一个标签页。我今天发现确实新打的包支持点开链接了，但是微信读书传书还是点不开
3. 如何配置打开自适应屏幕（最大铺满，但不是全屏）

**Reply by tw93** [reply]
好勒 懂了 有空我看看
